ingest/src/main/java/com/microsoft/azure/kusto/ingest/QueuedIngestClientImpl.java

```diff
@@ -50,14 +50,15 @@ public class QueuedIngestClientImpl extends IngestClientBase implements QueuedIn
     private final ResourceManager resourceManager;
     private final AzureStorageClient azureStorageClient;
 
-    QueuedIngestClientImpl(ConnectionStringBuilder csb, @Nullable HttpClientProperties properties) throws URISyntaxException {
-        this(csb, properties == null ? null : HttpClientFactory.create(properties));
+    QueuedIngestClientImpl(ConnectionStringBuilder csb, @Nullable HttpClientProperties properties, boolean autoCorrectEndpoint) throws URISyntaxException {
+        this(csb, properties == null ? null : HttpClientFactory.create(properties),autoCorrectEndpoint);
     }
 
-    QueuedIngestClientImpl(ConnectionStringBuilder csb, CloseableHttpClient httpClient) throws URISyntaxException {
+    QueuedIngestClientImpl(ConnectionStringBuilder csb, CloseableHttpClient httpClient, boolean autoCorrectEndpoint) throws URISyntaxException {
```

Comment:
Where is the second half of the PR? (Adding checks to not replace the url in certain cases)